src/main/resources/config/application-artemis.yml

```
@@ -109,3 +109,6 @@ artemis:
 
     plagiarism-checks:
         plagiarism-results-limit: 100
+
+    science:
+        enable-on-start: false # Set this to true to enable the feature toggle for the science module on startup.
```

```suggestion
    research:
        event-logging:
            enable: false
```
I think that an admin that did not read the PR here would have no idea what ‘enable science’ means.

The proposed config key would also allow for potential extension in the future, e.g.,
```yaml
research:
  # enable base functionality globally, then allow more fine-grained selection of events that are collected
  event-logging:
    enable: true
    lecure-events: true
    programming-exercise-events: false
  # some other research-functionality options here in the future?
```